lib/backend/helpers.go

```diff
@@ -61,3 +61,38 @@ func ReleaseLock(ctx context.Context, backend Backend, lockName string) error {
 	}
 	return nil
 }
+
+// ResetLockTTL resets the TTL on a given lock.
+func ResetLockTTL(ctx context.Context, backend Backend, lockName string) error {
+	if lockName == "" {
+		return trace.BadParameter("missing parameter lock name")
+	}
+
+	key := []byte(filepath.Join(locksPrefix, lockName))
+
+	item, err := backend.Get(ctx, key)
+	if err != nil {
+		return trace.Wrap(err)
+	}
+
+	if _, err := backend.Put(ctx, *item); err != nil {
```

Comment:
It looks that raw Get/Put operation doesn't refresh the TTL of an item because TTL is stored in an item attribute and GET/PUT doesn't refresh it.  
https://github.com/gravitational/teleport/blob/879f8c2b486809298ecddea842ce4f64de2e655f/lib/backend/dynamo/dynamodbbk.go#L805